src/backend/Normalizer/index.ts

```diff
@@ -81,8 +82,8 @@ export function normalizer(config: NormalizerConfig): void {
         subscribePromises.push(subscriber(config.topics[i]));
     }
 
-    function failureCb(error: Error): void {
-        logger.publishLog(LogLevels.ERROR, SERVICE_INSTANCE_ID, ': Failed ', error.message);
+    function failureCb(reason: unknown): void {
+        logger.publishLog(LogLevels.ERROR, SERVICE_INSTANCE_ID, ': Failed ', getErrorMessage(reason));
```

Comment:
I don't see what the problem is in publishing the entire error object?
if we pass in:
logger.publishLog(LogLevels.ERROR, SERVICE_INSTANCE_ID, ': Failed ', error); 
`the error object is stringified`

Reply:
Also, you might want to pull in the latest master on this branch (resolve the conflicts), before we can go for a merge

Reply:
@yashjain28 stringifying error objects doesn't work as expected in code services. Try this sample to see for yourself - 
```javascript
function testErrorStringify(req,resp){
    // These are parameters passed into the code service
    var params = req.params
    try {
        const thing = does.not.exist;
    } catch (e) {
        resp.error(JSON.stringify(e)) // returns {}
        resp.error(e.message); // returns 'identifier 'does' undefined'
    }
    resp.success("Success");
}
```

Reply:
ok :(
It's a javascript thing, just checked! 

I think `stack` would be more helpful for you to debug @savfaire I tried in a code service, it gave us the line number as well. You can add if you think it's helpful

function getErrorMessage(error: unknown): string {
    if (typeof error === 'object' && error.stack) {
        return error.stack;
    } else {
        return JSON.stringify(error);
    }
}

Reply:
FYI: stack is a string as well.

Reply:
stack is a "non-standard" item that is not guaranteed to be implemented. So I'm not entirely sure we would want to use it, since we integrate with third party libraries (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/stack).

Reply:
I think we would be better served if we always returned Error objects (new Error("my error message")) when rejecting promises.

Reply:
It's okay if it's not-standard, I think we can have a check if the `stack` property exists, we add it to the response. The reason being, if the stack is available, it's very valuable while debugging. 
Example stack when running a code service
```
ReferenceError: identifier 'does' undefined
    at [anon] (duk_js_var.c:1196) internal
    at testErrorStringify (eval:20)
    at eval (eval:28) preventsyield

```

Reply:
Ok, I've reworked the code so that promises are always rejected with an Error object. I also added error.stack to the getErrorMessage function.

Reply:
Figuring out how to fix the build error